cmds/contest-generator/README.md

```diff
@@ -0,0 +1,63 @@
+## ConTest Generator
+
+`contest-generator` is a tool used to build ConTest with the plugins you want.
+
+We provide a standard [main.go](/cmds/contest/main.go), where we only include
+the most common plugins.
+
+If you need to build `ConTest` with a custom set of plugins without maintaining
+your own `main.go` program, you can `contest-generator`.
+
+Just define your plugins in
+[core-plugins.txt](/cmds/contest-generator/core-plugins.txt) with the format
```

Comment:
core-plugins.txt -> .yml, here and in a few other places

Reply:
whoops, thanks

Reply:
done